eng/Versions.props

```diff
@@ -46,7 +46,7 @@
     <MicrosoftExtensionsDependencyInjectionPackageVersion>6.0.0-preview.3.21201.4</MicrosoftExtensionsDependencyInjectionPackageVersion>
     <!-- Microsoft.Extensions.FileProviders.Physical -->
     <MicrosoftExtensionsFileProvidersPhysicalPackageVersion>6.0.0-preview.3.21201.4</MicrosoftExtensionsFileProvidersPhysicalPackageVersion>
-    <NewtonsoftJsonPackageVersion>11.0.2</NewtonsoftJsonPackageVersion>
+    <NewtonsoftJsonPackageVersion>13.0.1</NewtonsoftJsonPackageVersion>
```

Comment:
Is this changing for scaffolding as well? That might have impact if VS is shipping another version. Can we confine this just to Identity?

Reply:
yup it is used by scaffolding a bunch. Reverted it for now but adding ticket to track the upgrade for next preview. 
#1550